README.md

```diff
@@ -96,7 +96,7 @@ Open a terminal in it and run the command below after adapting the `--from-versi
 npx -- activeui-migration migrate -i content.json -o migrated-content.json -s servers.json --from-version 4.3 --to-version 5.1
```

Comment:
"activeui" will still be used in the CLI. Do we want to change that eventually too ? Should I open a ticket ?

Reply:
I would say that we do, but that would be breaking, so I guess we should raise a 5.2 ticket.